NEWS

```diff
@@ -1,3 +1,7 @@
+4.4.0
+
+Extract passenger threads fix to the separate file
```

Comment:
add what the users need to do to remain "compatible" with previous code. Pretty much, how to adjust their code.

Reply:
they will need to add this file: https://github.com/splitio/ruby-client/blob/feature/move_passenger_threads_fix/Detailed-README.md#phusion-passenger